spec/lib/synapse/haproxy_spec.rb

```diff
@@ -208,6 +276,7 @@ class MockWatcher; end;
       expect(haproxy.opts['do_writes']).to eql(true)
       expect(haproxy.opts['do_socket']).to eql(true)
       expect(haproxy.opts['do_reloads']).to eql(true)
+      expect(haproxy.opts['use_nerve_weights']).to eql(nil)
```

Comment:
what is the expected default value?

Reply:
The expected default value, which is how people would be executing before this change, is to have the flag not exist.